content/Watsonx/Assistant/104.md

```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
+
 ### Adding data to the Analyze page via Lendyr
 
-Open your assistant on the Lendyr using the link you created in the previous section to embed your assistant. Try out a few different conversations. To engage in an additional separate conversation with your assistant there should be a **Restart** button in upper-left menu.
+Conversation data is added to the Analytics page when your assistant engages in conversation with end users.
+
+Open your assistant on the Lendyr website (consult the **Preview your assistant on Lendyr Bank’s website** portion of section 2. Actions to open your assistant on the Lendyr website if you do not know how to do this) and try out a few conversations. To engage in an additional separate conversation with your assistant, you can restart the assistant by opening your custom Lendyr URL in a new tab or by following these instructions:
+
+1. Open a **Guided Tour** tile from the splash screen on your custom Lendyr.com website.
+
 
 ![](./images/104/image-001.jpg)
 
+2. Follow the instructions to complete the tour. Then, type **RESTART_SESSION (1)** to the virtual assistant **(2)**. (This is same as clicking the **Refresh icon** in the watsonx Assistant’s Preview window.)
+
 ![](./images/104/image-002.jpg)
 
-After this return to the **Analyze** page. Click the **Refresh** button in the top-right corner to pull in the latest data from the conversation you just had with the assistant.
+**Pro tip:** Restarting your assistant's session on the Lendyr website is also useful when the assistant doesn’t reflect the latest changes you have saved in the build experience. In other words, restart the assistant’s session if you need to force a sync to pull in your latest changes.
+
+Now return to the Analyze page. Click the **Refresh** button in the top right corner of the Analyze
+page to pull in the latest data from the conversations you just had with the assistant.
+
 
 ![](./images/104/image-003.jpg)
 
 When the page refreshes, you should now see a slightly higher request and conversation count.
 
 ### Adding data to the Analyze page via Preview
 
-Imagine you are a builder who has finished writing a set of actions that address a new use case.
+Imagine you are a builder who has finished writing a set of actions that address a new use case. Or imagine you are a product manager overseeing a virtual assistant project, and a subject-matter expert has just let you know that they fine-tuned a few conversational flows in the assistant.
 
-You need a way to share your assistant with a testing or quality assurance (QA) team, so they can go test the updated actions before you deploy them to a live or production environment. In order to do so:
+
+You need a way to share your assistant with a testing or quality assurance (QA) team, so they can go test the updated actions before you deploy them to a live or production environment. 
 
 1. Go the the **Preview** page for your assistant.
```

Comment:
Include event numbers for Preview and copy link to share.

Reply:
added